docs/developer/contributing/development-functional-tests.asciidoc

```diff
@@ -6,7 +6,19 @@ We use functional tests to make sure the {kib} UI works as expected. It replaces
 [discrete]
 === Running functional tests
 
-The `FunctionalTestRunner` (FTR) is very bare bones and gets most of its functionality from its config file. The {kib} repo contains many FTR config files which use slightly different configurations for the {kib} server or {es}, have different test files, and potentially other config differences. You can find a manifest of all the FTR config files in `.buildkite/ftr_configs.yml`. If you’re writing a plugin outside the {kib} repo, you will have your own config file.
+The `FunctionalTestRunner` (FTR) is very bare bones and gets most of its functionality from its config file. The {kib} repo contains many FTR config files which use slightly different configurations for the {kib} server or {es}, have different test files, and potentially other config differences.
+FTR config files are organised in manifest files, based on testing area and type of distribution:
+serverless:
+- ftr_base_serverless_configs.yml
+- ftr_oblt_serverless_configs.yml
+- ftr_security_serverless_configs.yml
+- ftr_search_serverless_configs.yml
+stateful:
+- ftr_platform_stateful_configs.yml
+- ftr_oblt_stateful_configs.yml
+- ftr_security_stateful_configs.yml
+- ftr_search_stateful_configs.yml
+If you’re writing a plugin outside the {kib} repo, you will have your own config file.
```

Comment:
@pheyos This is the only place where we mentioned original manifest file. Do you think we need to add more context here or put this info in other place?